frontend/src/components/HacknightsParticipants.vue

```diff
@@ -65,6 +65,9 @@
               <v-icon>mdi-account-outline</v-icon>
             </v-list-item-avatar>
             <v-list-item-title v-text="item.github"></v-list-item-title>
+            <v-btn v-on:click="onDeleteParticipant(item)">
+              <i class="fas fa-user-times fa-lg"></i>
```

Comment:
Maybe we could also use red color to communicate deleteing function?
```suggestion
              <i class="button_delete fas fa-user-times fa-lg"></i>
```
Then we could add new class:
```
.button_delete {
  color: $red;
}
```
And add global red (e.g. #ed130c) in `scss/colors.scss` so it can parse it from a global source :)